### DISCUSSION TITLE
コンソールウィンドウが無限ループ

### DISCUSSION THREAD

**Discussion by mitkeicat** [sotanengel/rustlings-jp#6] [Q&A] [answered]
コメントのほうに書いてしまいましたが、あちらは不具合報告ではないので、こちらに書き直します。
Rustのダウンロードとこの練習問題のインストールには成功し、ウェルカム画面も表示されました。そのタイミングで、
エンターキーを押して練習問題が始まるかな、というタイミングで、コンソールウィンドウが画面の表示が止まらなくなり、無限ループを繰り返している状態になります。当然、練習問題はできないです。なにかの設定が必要なのでしょうか？
コンソールはパワーシェルの7.4.5 ですが、他のシェルでも同じような不具合が起きます。
OSはWindows10 Pro　２２H2,　rustc は1.81.0 になります。
よろしくお願いします。

**Comment by sotanengel** [comment, marked as answer]
@mitkeicat ご連絡ありがとうございます。
私の端末で改めて最新バージョンのフォルダを起動してみたのですが、いただいたバグは再現できないですね。。。。
<img width="1108" alt="image" src="https://github.com/user-attachments/assets/f8966cdd-5a73-4592-9b8b-94e799532805">

もし可能であれば以下のコマンドを使って、本家のrustlingsが起動可能か確かめていただくことはできますか？？
もしも以下を実行しても、同じようなバグが発生する場合には本家のコード側に何らかの問題が発生している可能性があります🙇

```
rustlings init

cd rustlings/

rustlings
```

**Reply by sotanengel** [reply]
もしも上記のコマンドで何もエラーが起きない場合には、
私のコード(or サードパーティ用の構成)特有のエラーだと思いますので、
お手数ではありますが、本家の*exercises*配下のフォルダを全削除して、rustlings-jpの*exercises*のフォルダを全部移植いただけますか？(問題文自体が日本語で読めますので)

**Reply by mitkeicat** [reply]
sotanengel  様、対応のほどありがとうございます。
上の コマンド試してみました。結果的に、コンソールの中で無限ループは発生します。

しかし、パワーシェルのウインドウを、右上のボタンを押して全画面表示にすると、正常に画面が表示されるようになりました。
ウィンドウのサイズを小さくすると再び画面がチラツクような動作になります。

どうしてかわかりませんが、とりあえず、動作はできるようになりましたので、これで練習してみます。
ありがとうございました。
